build/Common.nonprod.props

```diff
@@ -36,7 +36,7 @@
     <MicrosoftExtensionsLoggingPkgVer>[5.0.0,6.0)</MicrosoftExtensionsLoggingPkgVer>
     <MicrosoftExtensionsLoggingAbstractionsPkgVer>[5.0.0,6.0)</MicrosoftExtensionsLoggingAbstractionsPkgVer>
     <MicrosoftExtensionsLoggingConsolePkgVer>[5.0.0,6.0)</MicrosoftExtensionsLoggingConsolePkgVer>
-    <MicrosoftNETTestSdkPkgVer>[16.9.1]</MicrosoftNETTestSdkPkgVer>
+    <MicrosoftNETTestSdkPkgVer>[16.10.0]</MicrosoftNETTestSdkPkgVer>
```

Comment:
can you put a comment above this line to remind about updating the codecoverage.ps1 file as well when updating this?

Reply:
I have added a comment in the `Common.props` file where we have the PackageReference for `Microsoft.CodeCoverage`

Reply:
Is it possible to update PowerShell code to "fetch" the version from the `Common.props` instead of hardcode?